src/Console/Command/CleanCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Lmc\Steward\Console\Command;
+
+use Lmc\Steward\Console\CommandEvents;
+use Lmc\Steward\Console\Event\BasicConsoleEvent;
+use Lmc\Steward\Publisher\XmlPublisher;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+/**
+ * Clean previous log files in the logs directory
+ */
+class CleanCommand extends Command
+{
+    /** @var Filesystem */
+    private $filesystem;
+
+    /**
+     * @param Filesystem $filesystem
+     */
+    public function setFilesystem(Filesystem $filesystem)
+    {
+        $this->filesystem = $filesystem;
+    }
+
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        $this->setName('clean')
+            ->setDescription('Clean logs directory')
+            ->addOption(
+                RunCommand::OPTION_LOGS_DIR,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Path to directory with logs',
+                STEWARD_BASE_DIR . '/logs'
+            );
+
+        $this->getDispatcher()->dispatch(CommandEvents::CONFIGURE, new BasicConsoleEvent($this));
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int|null
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $defaultLogsDir = $this->getDefinition()->getOption(RunCommand::OPTION_LOGS_DIR)->getDefault();
+        $logsDir = $input->getOption(RunCommand::OPTION_LOGS_DIR);
+
+        // If default path to logs directory is used and it does not exist, create the directory
+        if ($logsDir == $defaultLogsDir) {
```

Comment:
Why auto-create only default logs dir? Even non-default logs dir is cleaned, so why not auto-create them also?